<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fso::fred::dialogs::VariableEditorDialog</class>
 <widget class="QDialog" name="fso::fred::dialogs::VariableEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>649</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fiction Viewer Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Variables and Containers Editor</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>380</height>
        </size>
       </property>
       <property name="title">
        <string>Containers</string>
       </property>
       <widget class="QTableWidget" name="containersTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>231</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>341</width>
          <height>191</height>
         </rect>
        </property>
        <property name="title">
         <string>Contents</string>
        </property>
        <widget class="QTableWidget" name="containerContentsTable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>221</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>30</y>
           <width>91</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="addContainerItemButton">
            <property name="text">
             <string>Add Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="copyContainerItemButton">
            <property name="text">
             <string>Copy Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteContainerItemButton">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Delete Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>40</y>
          <width>97</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="addContainerButton">
           <property name="text">
            <string>Add Container</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyContainerButton">
           <property name="text">
            <string>Copy Container</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteContainerButton">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Delete Container</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>20</y>
          <width>211</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Persistence Options</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>201</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QRadioButton" name="saveContainerOnMissionCloseRadio">
            <property name="text">
             <string>Save on Mission Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="saveContainerOnMissionCompletedRadio">
            <property name="text">
             <string>Save on Mission Completed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="setContainerAsEternalCheckbox">
            <property name="text">
             <string>Eternal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="networkContainerCheckbox">
            <property name="text">
             <string>Network-Variable</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>170</y>
          <width>211</width>
          <height>191</height>
         </rect>
        </property>
        <property name="title">
         <string>Type Options</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>201</width>
           <height>152</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Container Type</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="setContainerAsListRadio">
              <property name="text">
               <string>List</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="setContainerAsMapRadio">
              <property name="text">
               <string>Map</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Key Type</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="setContainerKeyAsNumberRadio">
              <property name="text">
               <string>Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="setContainerKeyAsStringRadio">
              <property name="text">
               <string>String</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Data Type</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="setContainerAsNumberRadio">
              <property name="text">
               <string>Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="setContainerAsStringRadio">
              <property name="text">
               <string>String</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>230</height>
        </size>
       </property>
       <property name="title">
        <string>Variables</string>
       </property>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>100</y>
          <width>311</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Options</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>300</width>
           <height>101</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="setVariableAsStringRadio">
              <property name="text">
               <string>String</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="setVariableAsNumberRadio">
              <property name="text">
               <string>Number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="networkVariableCheckbox">
              <property name="text">
               <string>Network-Variable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="saveVariableOnMissionCloseRadio">
              <property name="text">
               <string>Save on Mission Close</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="saveVariableOnMissionCompletedRaio">
              <property name="text">
               <string>Save on Mission Completed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="setVariableAsEternalcheckbox">
              <property name="text">
               <string>Eternal</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QTableWidget" name="variablesTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>241</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>38</y>
          <width>301</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="addVariableButton">
           <property name="text">
            <string>Add Variable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyVariableButton">
           <property name="text">
            <string>Copy Variable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteVariableButton">
           <property name="text">
            <string>Delete Variable</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
